src/components/EmptyCharacterSheetSlot/index.tsx

```diff
@@ -95,7 +96,8 @@ const EmptySlotTypeIcons = ({ slotTypes }: { slotTypes: CardSlotMetadata["slotTy
                 height: "42px",
                 width: "42px",
                 opacity: "70%",
-                borderRadius: "2px",
+                borderRadius: "8px",
+                border: "1px solid black",
```

Comment:
i think there is too much border radius here (we usually use 2px i think) and the border is a bit too harsh. I would like to try a box shadow type approach as we use in the health/energy/gold icons at the bottom which is a bit more subtle but doesnt block this PR

Reply:
Yep sure, I wanted to try and add more distinction between these empty slots, but don't feel too strongly about this